### ISSUE TITLE
Speech to Text recognized  but neither audio responce or intent handling

### ISSUE THREAD

**Issue by Chreece** [rhasspy/rhasspy#126]
Hello there!
Thank you for reading my issue.
Shout out to all that contribute to build the best free and offline-privacy voice assistant!
You are doing a very good Job! Thank you and keep up!!!

I have a problem. The rhasspy seems to start listening (when i say the wake word i got the sound alert that listens), but after saying my command i don't get any feedback. Neither the sound for recorded WAV or the result i wanted for the intent.
After looking the logs i noticed that it recognises what i say (at the end of the logs "τι ώρα είναι" but it doesn't send the event to Home Assistant. 
If I play the recording in rhasspy i hear my command with a long recording after that.
Trying the same intent "τι ώρα είναι" on the Recognize field in rhasspy, it works and home assistant gets the event.
Hitting Wake Up and giving vocally the command, i got a time out. 
Is it me or a bug?
I have tried: Restart, Train. Docker restart rhasspy.
Also i noticed that the last command in logs is "Ignoring unknown session default-picovoice-xxxxxxxxxxxxxxx" could that be a lead?

Thank you for your time stranger!

The LOGS:

```txt
[DEBUG:2020-11-03 12:50:45,618] rhasspywake_porcupine_hermes: -> HotwordDetected(model_id='/usr/lib/rhasspy/rhasspy-wake-porcupine-hermes/rhasspywake_porcupine_hermes/porcupine/resources/keyword_files/raspberrypi/picovoice.ppn', model_version='', model_type='personal', current_sensitivity=0.5, site_id='default', session_id=None, send_audio_captured=None, lang=None)
[DEBUG:2020-11-03 12:50:45,619] rhasspywake_porcupine_hermes: Publishing 303 bytes(s) to hermes/hotword/picovoice/detected
[DEBUG:2020-11-03 12:50:45,623] rhasspyserver_hermes: <- HotwordDetected(model_id='/usr/lib/rhasspy/rhasspy-wake-porcupine-hermes/rhasspywake_porcupine_hermes/porcupine/resources/keyword_files/raspberrypi/picovoice.ppn', model_version='', model_type='personal', current_sensitivity=0.5, site_id='default', session_id=None, send_audio_captured=None, lang=None)
[DEBUG:2020-11-03 12:50:45,626] rhasspydialogue_hermes: <- HotwordDetected(model_id='/usr/lib/rhasspy/rhasspy-wake-porcupine-hermes/rhasspywake_porcupine_hermes/porcupine/resources/keyword_files/raspberrypi/picovoice.ppn', model_version='', model_type='personal', current_sensitivity=0.5, site_id='default', session_id=None, send_audio_captured=None, lang=None)
[DEBUG:2020-11-03 12:50:45,627] rhasspydialogue_hermes: Playing WAV /usr/lib/rhasspy/etc/wav/beep_hi.wav
[DEBUG:2020-11-03 12:50:45,629] rhasspydialogue_hermes: -> HotwordToggleOff(site_id='default', reason=<HotwordToggleReason.PLAY_AUDIO: 'playAudio'>)
[DEBUG:2020-11-03 12:50:45,629] rhasspydialogue_hermes: Publishing 44 bytes(s) to hermes/hotword/toggleOff
[DEBUG:2020-11-03 12:50:45,632] rhasspydialogue_hermes: -> AsrToggleOff(site_id='default', reason=<AsrToggleReason.PLAY_AUDIO: 'playAudio'>)
[DEBUG:2020-11-03 12:50:45,632] rhasspydialogue_hermes: Publishing 44 bytes(s) to hermes/asr/toggleOff
[DEBUG:2020-11-03 12:50:45,633] rhasspydialogue_hermes: -> AudioPlayBytes(83948 byte(s)) to hermes/audioServer/default/playBytes/7e7eb3ec-a2a9-469a-aa84-173feab4a45d
[DEBUG:2020-11-03 12:50:45,634] rhasspydialogue_hermes: Waiting for playFinished (id=7e7eb3ec-a2a9-469a-aa84-173feab4a45d, timeout=1.2012925170068027)
[DEBUG:2020-11-03 12:50:45,633] rhasspywake_porcupine_hermes: <- HotwordToggleOff(site_id='default', reason=<HotwordToggleReason.PLAY_AUDIO: 'playAudio'>)
[DEBUG:2020-11-03 12:50:45,634] rhasspywake_porcupine_hermes: Disabled
[DEBUG:2020-11-03 12:50:45,638] rhasspyspeakers_cli_hermes: <- AudioPlayBytes(83948 byte(s))
[DEBUG:2020-11-03 12:50:45,639] rhasspyspeakers_cli_hermes: ['aplay', '-q', '-t', 'wav', '-D', 'default:CARD=Headphones']
[DEBUG:2020-11-03 12:50:45,663] rhasspyasr_pocketsphinx_hermes: <- AsrToggleOff(site_id='default', reason=<AsrToggleReason.PLAY_AUDIO: 'playAudio'>)
[DEBUG:2020-11-03 12:50:45,664] rhasspyasr_pocketsphinx_hermes: Disabled
[DEBUG:2020-11-03 12:50:46,171] rhasspyspeakers_cli_hermes: -> AudioPlayFinished(id='7e7eb3ec-a2a9-469a-aa84-173feab4a45d', session_id='7e7eb3ec-a2a9-469a-aa84-173feab4a45d')
[DEBUG:2020-11-03 12:50:46,171] rhasspyspeakers_cli_hermes: Publishing 99 bytes(s) to hermes/audioServer/default/playFinished
[DEBUG:2020-11-03 12:50:46,174] rhasspydialogue_hermes: <- AudioPlayFinished(id='7e7eb3ec-a2a9-469a-aa84-173feab4a45d', session_id='7e7eb3ec-a2a9-469a-aa84-173feab4a45d')
[DEBUG:2020-11-03 12:50:46,174] rhasspytts_cli_hermes: <- AudioPlayFinished(id='7e7eb3ec-a2a9-469a-aa84-173feab4a45d', session_id='7e7eb3ec-a2a9-469a-aa84-173feab4a45d')
[DEBUG:2020-11-03 12:50:46,175] rhasspydialogue_hermes: -> HotwordToggleOn(site_id='default', reason=<HotwordToggleReason.PLAY_AUDIO: 'playAudio'>)
[DEBUG:2020-11-03 12:50:46,175] rhasspydialogue_hermes: Publishing 44 bytes(s) to hermes/hotword/toggleOn
[DEBUG:2020-11-03 12:50:46,177] rhasspydialogue_hermes: -> AsrToggleOn(site_id='default', reason=<AsrToggleReason.PLAY_AUDIO: 'playAudio'>)
[DEBUG:2020-11-03 12:50:46,177] rhasspydialogue_hermes: Publishing 44 bytes(s) to hermes/asr/toggleOn
[WARNING:2020-11-03 12:50:46,177] rhasspydialogue_hermes: No session for id default-picovoice-2f7e83c3-a506-48d0-a4ee-7dcb125bd928
[DEBUG:2020-11-03 12:50:46,178] rhasspydialogue_hermes: Handling queued session
[DEBUG:2020-11-03 12:50:46,178] rhasspydialogue_hermes: Session aborted: default-picovoice-2f7e83c3-a506-48d0-a4ee-7dcb125bd928
[WARNING:2020-11-03 12:50:46,178] rhasspydialogue_hermes: No session for id default-picovoice-2f7e83c3-a506-48d0-a4ee-7dcb125bd928
[DEBUG:2020-11-03 12:50:46,178] rhasspywake_porcupine_hermes: <- HotwordToggleOn(site_id='default', reason=<HotwordToggleReason.PLAY_AUDIO: 'playAudio'>)
[DEBUG:2020-11-03 12:50:46,178] rhasspywake_porcupine_hermes: Enabled
[DEBUG:2020-11-03 12:50:46,179] rhasspydialogue_hermes: -> HotwordToggleOn(site_id='default', reason=<HotwordToggleReason.DIALOGUE_SESSION: 'dialogueSession'>)
[DEBUG:2020-11-03 12:50:46,179] rhasspyasr_pocketsphinx_hermes: <- AsrToggleOn(site_id='default', reason=<AsrToggleReason.PLAY_AUDIO: 'playAudio'>)
[DEBUG:2020-11-03 12:50:46,180] rhasspyasr_pocketsphinx_hermes: Enabled
[DEBUG:2020-11-03 12:50:46,179] rhasspydialogue_hermes: Publishing 50 bytes(s) to hermes/hotword/toggleOn
[DEBUG:2020-11-03 12:50:46,180] rhasspydialogue_hermes: Starting new session (id=default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e)
[DEBUG:2020-11-03 12:50:46,181] rhasspydialogue_hermes: -> DialogueSessionStarted(session_id='default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e', site_id='default', custom_data='picovoice', lang=None)
[DEBUG:2020-11-03 12:50:46,182] rhasspywake_porcupine_hermes: <- HotwordToggleOn(site_id='default', reason=<HotwordToggleReason.DIALOGUE_SESSION: 'dialogueSession'>)
[DEBUG:2020-11-03 12:50:46,182] rhasspywake_porcupine_hermes: Enabled
[DEBUG:2020-11-03 12:50:46,182] rhasspydialogue_hermes: Publishing 133 bytes(s) to hermes/dialogueManager/sessionStarted
[DEBUG:2020-11-03 12:50:46,185] rhasspydialogue_hermes: -> HotwordToggleOff(site_id='default', reason=<HotwordToggleReason.DIALOGUE_SESSION: 'dialogueSession'>)
[DEBUG:2020-11-03 12:50:46,185] rhasspydialogue_hermes: Publishing 50 bytes(s) to hermes/hotword/toggleOff
[DEBUG:2020-11-03 12:50:46,185] rhasspydialogue_hermes: Listening for session default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e
[DEBUG:2020-11-03 12:50:46,185] rhasspywake_porcupine_hermes: Receiving audio
[DEBUG:2020-11-03 12:50:46,187] rhasspyasr_pocketsphinx_hermes: Receiving audio
[DEBUG:2020-11-03 12:50:46,187] rhasspydialogue_hermes: -> AsrStartListening(site_id='default', session_id='default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e', lang=None, stop_on_silence=True, send_audio_captured=True, wakeword_id='picovoice', intent_filter=None)
[DEBUG:2020-11-03 12:50:46,187] rhasspydialogue_hermes: Publishing 205 bytes(s) to hermes/asr/startListening
[DEBUG:2020-11-03 12:50:46,233] rhasspywake_porcupine_hermes: <- HotwordToggleOff(site_id='default', reason=<HotwordToggleReason.DIALOGUE_SESSION: 'dialogueSession'>)
[DEBUG:2020-11-03 12:50:46,234] rhasspyasr_pocketsphinx_hermes: <- AsrStartListening(site_id='default', session_id='default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e', lang=None, stop_on_silence=True, send_audio_captured=True, wakeword_id='picovoice', intent_filter=None)
[DEBUG:2020-11-03 12:50:46,234] rhasspywake_porcupine_hermes: Disabled
[DEBUG:2020-11-03 12:50:46,234] rhasspyasr_pocketsphinx_hermes: Starting listening (session_id=default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e)
[DEBUG:2020-11-03 12:50:46,249] rhasspyasr_pocketsphinx_hermes: Receiving audio
[ERROR:2020-11-03 12:51:16,214] rhasspydialogue_hermes: Session timed out for site default: default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e
[DEBUG:2020-11-03 12:51:16,215] rhasspydialogue_hermes: -> AsrStopListening(site_id='default', session_id='default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e')
[DEBUG:2020-11-03 12:51:16,215] rhasspydialogue_hermes: Publishing 92 bytes(s) to hermes/asr/stopListening
[DEBUG:2020-11-03 12:51:16,217] rhasspydialogue_hermes: -> DialogueSessionEnded(termination=DialogueSessionTermination(reason=<DialogueSessionTerminationReason.TIMEOUT: 'timeout'>), session_id='default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e', site_id='default', custom_data='picovoice')
[DEBUG:2020-11-03 12:51:16,218] rhasspydialogue_hermes: Publishing 157 bytes(s) to hermes/dialogueManager/sessionEnded
[DEBUG:2020-11-03 12:51:16,219] rhasspydialogue_hermes: -> HotwordToggleOn(site_id='default', reason=<HotwordToggleReason.DIALOGUE_SESSION: 'dialogueSession'>)
[DEBUG:2020-11-03 12:51:16,219] rhasspydialogue_hermes: Publishing 50 bytes(s) to hermes/hotword/toggleOn
[DEBUG:2020-11-03 12:51:16,233] rhasspywake_porcupine_hermes: <- HotwordToggleOn(site_id='default', reason=<HotwordToggleReason.DIALOGUE_SESSION: 'dialogueSession'>)
[DEBUG:2020-11-03 12:51:16,233] rhasspyasr_pocketsphinx_hermes: <- AsrStopListening(site_id='default', session_id='default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e')
[DEBUG:2020-11-03 12:51:16,235] rhasspywake_porcupine_hermes: Enabled
[DEBUG:2020-11-03 12:51:16,240] rhasspyasr_pocketsphinx_hermes: Received a total of 979056 byte(s) for WAV data for session default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e
[DEBUG:2020-11-03 12:51:16,240] rhasspyasr_pocketsphinx_hermes: Transcribing 954284 byte(s) of audio data
[DEBUG:2020-11-03 12:51:16,251] rhasspywake_porcupine_hermes: Receiving audio
INFO: cmn.c(133): CMN: 43.75 11.26 -11.05 22.73 -2.17 -3.94 -16.47 -6.27  7.07 -14.84 -2.59  4.82 -0.40
INFO: ngram_search_fwdtree.c(1550):     1263 words recognized (13/fr)
INFO: ngram_search_fwdtree.c(1552):   360885 senones evaluated (3609/fr)
INFO: ngram_search_fwdtree.c(1556):  2315386 channels searched (23153/fr), 64879 1st, 31209 last
INFO: ngram_search_fwdtree.c(1559):     2229 words for which last channels evaluated (22/fr)
INFO: ngram_search_fwdtree.c(1561):   228234 candidate words for entering last phone (2282/fr)
INFO: ngram_search_fwdtree.c(1564): fwdtree 4.94 CPU 4.943 xRT
INFO: ngram_search_fwdtree.c(1567): fwdtree 5.03 wall 5.030 xRT
INFO: ngram_search_fwdflat.c(302): Utterance vocabulary contains 63 words
INFO: ngram_search_fwdflat.c(948):      744 words recognized (7/fr)
INFO: ngram_search_fwdflat.c(950):    64212 senones evaluated (642/fr)
INFO: ngram_search_fwdflat.c(952):    55602 channels searched (556/fr)
INFO: ngram_search_fwdflat.c(954):     3851 words searched (38/fr)
INFO: ngram_search_fwdflat.c(957):     2369 word transitions (23/fr)
INFO: ngram_search_fwdflat.c(960): fwdflat 0.43 CPU 0.435 xRT
INFO: ngram_search_fwdflat.c(963): fwdflat 0.44 wall 0.437 xRT
[DEBUG:2020-11-03 12:51:21,711] rhasspyasr_pocketsphinx.transcribe: Decoded audio in 5.466955737007083 second(s)
INFO: ngram_search.c(1250): lattice start node <s>.0 end node </s>.77
INFO: ngram_search.c(1276): Eliminated 0 nodes before end node
INFO: ngram_search.c(1381): Lattice has 96 nodes, 101 links
INFO: ps_lattice.c(1374): Bestpath score: -2782
INFO: ps_lattice.c(1378): Normalizer P(O) = alpha(</s>:77:98) = -193858
INFO: ps_lattice.c(1435): Joint P(O,S) = -198415 P(S|O) = -4557
INFO: ngram_search.c(872): bestpath 0.00 CPU 0.001 xRT
INFO: ngram_search.c(875): bestpath 0.00 wall 0.001 xRT
INFO: ngram_search.c(1027): bestpath 0.00 CPU 0.000 xRT
INFO: ngram_search.c(1030): bestpath 0.00 wall 0.000 xRT
[DEBUG:2020-11-03 12:51:21,714] rhasspyasr_pocketsphinx_hermes: Transcription(text='τι ώρα είναι', likelihood=0.63397051780544, transcribe_seconds=5.466955737007083, wav_seconds=29.82, tokens=[TranscriptionToken(token='<s>', start_time=0.0, end_time=0.03, likelihood=0.9997999968180895), TranscriptionToken(token='τι', start_time=0.04, end_time=0.2, likelihood=0.9592487791926002), TranscriptionToken(token='ώρα', start_time=0.21, end_time=0.42, likelihood=0.992229443864552), TranscriptionToken(token='είναι', start_time=0.43, end_time=0.76, likelihood=0.9998999934083855), TranscriptionToken(token='</s>', start_time=0.77, end_time=0.98, likelihood=1.0)])
[DEBUG:2020-11-03 12:51:21,720] rhasspyasr_pocketsphinx_hermes: -> AsrTextCaptured(text='τι ώρα είναι', likelihood=0.63397051780544, seconds=5.466955737007083, site_id='default', session_id='default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e', wakeword_id=None, asr_tokens=[[AsrToken(value='<s>', confidence=0.9997999968180895, range_start=0, range_end=4, time=AsrTokenTime(start=0.0, end=0.03)), AsrToken(value='τι', confidence=0.9592487791926002, range_start=4, range_end=7, time=AsrTokenTime(start=0.04, end=0.2)), AsrToken(value='ώρα', confidence=0.992229443864552, range_start=7, range_end=11, time=AsrTokenTime(start=0.21, end=0.42)), AsrToken(value='είναι', confidence=0.9998999934083855, range_start=11, range_end=17, time=AsrTokenTime(start=0.43, end=0.76)), AsrToken(value='</s>', confidence=1.0, range_start=17, range_end=22, time=AsrTokenTime(start=0.77, end=0.98))]], lang=None)
[DEBUG:2020-11-03 12:51:21,720] rhasspyasr_pocketsphinx_hermes: Publishing 828 bytes(s) to hermes/asr/textCaptured
[DEBUG:2020-11-03 12:51:21,721] rhasspyasr_pocketsphinx_hermes: -> AsrAudioCaptured(954284 byte(s)) to rhasspy/asr/default/default/audioCaptured
[DEBUG:2020-11-03 12:51:21,728] rhasspyasr_pocketsphinx_hermes: Stopping listening (session_id=default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e)
[DEBUG:2020-11-03 12:51:21,740] rhasspydialogue_hermes: <- AsrTextCaptured(text='τι ώρα είναι', likelihood=0.63397051780544, seconds=5.466955737007083, site_id='default', session_id='default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e', wakeword_id=None, asr_tokens=[[AsrToken(value='<s>', confidence=0.9997999968180895, range_start=0, range_end=4, time=AsrTokenTime(start=0.0, end=0.03)), AsrToken(value='τι', confidence=0.9592487791926002, range_start=4, range_end=7, time=AsrTokenTime(start=0.04, end=0.2)), AsrToken(value='ώρα', confidence=0.992229443864552, range_start=7, range_end=11, time=AsrTokenTime(start=0.21, end=0.42)), AsrToken(value='είναι', confidence=0.9998999934083855, range_start=11, range_end=17, time=AsrTokenTime(start=0.43, end=0.76)), AsrToken(value='</s>', confidence=1.0, range_start=17, range_end=22, time=AsrTokenTime(start=0.77, end=0.98))]], lang=None)
[DEBUG:2020-11-03 12:51:21,756] rhasspyserver_hermes: Handling AsrTextCaptured (topic=hermes/asr/textCaptured, id=319a950c-f31f-4087-8a7f-84ae4e52f9e2)
[WARNING:2020-11-03 12:51:21,741] rhasspydialogue_hermes: Ignoring unknown session default-picovoice-2319a1c2-e730-4183-9f45-b4127464112e
p
```

My configuration:
```txt
{
    "dialogue": {
        "system": "rhasspy"
    },
    "handle": {
        "system": "hass"
    },
    "home_assistant": {
        "access_token": "mytoken",
        "handle_type": "event",
        "url": "http://myhomeassistantip:port"
    },
    "intent": {
        "system": "fsticuffs"
    },
    "microphone": {
        "arecord": {
            "device": "default:CARD=Microphone"
        },
        "system": "arecord"
    },
    "sounds": {
        "aplay": {
            "device": "default:CARD=Headphones"
        },
        "system": "aplay"
    },
    "speech_to_text": {
        "pocketsphinx": {
            "mix_weight": "0.5"
        },
        "system": "pocketsphinx"
    },
    "text_to_speech": {
        "espeak": {
            "voice": "el"
        },
        "system": "espeak",
        "wavenet": {
            "sample_rate": "44100"
        }
    },
    "wake": {
        "porcupine": {
            "keyword_path": "picovoice.ppn",
            "sensitivity": "0.5"
        },
        "system": "porcupine"
    }
}
```

**Comment by Terramoto** [comment]
I'm a noob here so take it with a handful of salt, but i wonder if this is the cause?
`resources/keyword_files/raspberrypi/picovoice.ppn', model_version='', model_type='personal', current_sensitivity=0.5, site_id='default', session_id=None, send_audio_captured=None, lang=None)`

**Comment by benedikt-bartscher** [comment]
Does rhasspy recognize an Intent? Just open the rhasspy dashboard and watch out for your intent or the "intent not recognized" message.

**Comment by Chreece** [comment]
So, I have managed to find the source of the problem.
I have an aquarium with a pump making a buzz noise in an constant frequency (it's the ac/dc transformer).
When my microphone is close to it, and the wake-word is detected (no problem with the Porcupine) then the Pocketsphinx seems to be confused and thinking someones speaking the entire time and gives an timeout error.
The same problem I've noticed when my laundry machine is working in a high rpm producing also a high pitch frequency.
I hope, I made myself clear :)
